app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __repr__(self) -> str:
+        return f'{{Name: {self.name}, ' \
+               f'Health: {self.health}, ' \
+               f'Hidden: {self.hidden}}}'
+
+    def __init__(self, name: str, health=100, hidden=False):
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
```

Comment:
init should be the first method of a class

Reply:
Thank you. I somehow overlooked this.